connectors/src/connectors/zendesk/index.ts

```diff
@@ -280,8 +275,70 @@ export class ZendeskConnectorManager extends BaseConnectorManager<null> {
     internalIds: string[];
     viewType: ContentNodesViewType;
   }): Promise<Result<ContentNode[], Error>> {
-    logger.info({ internalIds }, "Retrieving batch content nodes");
-    throw new Error("Method not implemented.");
+    const brandIds: number[] = [];
+    const brandHelpCenterIds: number[] = [];
+    const brandTicketsIds: number[] = [];
+    const categoryIds: number[] = [];
+    internalIds.forEach((internalId) => {
+      const { type, objectId } = getIdFromInternalId(
+        this.connectorId,
+        internalId
+      );
+      switch (type) {
+        case "brand": {
+          brandIds.push(objectId);
+          return;
+        }
+        case "tickets": {
+          brandTicketsIds.push(objectId);
+          return;
+        }
+        case "help-center": {
+          brandHelpCenterIds.push(objectId);
+          return;
+        }
+        case "category": {
+          categoryIds.push(objectId);
+          return;
+        }
+        default: {
+          return;
+        }
```

Comment:
Same here. Are we sure we'll never get article here ?

Reply:
handled this case specifically!